Reading_files/vascular.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>1) Please use a more descriptive and serious header. "Huge" is not a thing, and sample wise is written "sample-wise".</p><p>2) Each section should be a bullet point.</p><p>3) You also need to add when the notebook was created and when was the last time it was modified. </p><p> </p><p>For example:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">### Notebook for the concatenation of multiple anndata files into a single one for cardiac pericytes.

- **Developed by**: Srivalli Kolla
- **Würzburgh Institute for Systems Immunology - JMU Würzburg**
- **Created on**: 240408
- **Last modified**: 240408
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>